# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/Selection.h
# Commit: 4f30d7afdf22
# Full Hash: 4f30d7afdf22c652481513564b60b37053c0948b
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-05-08 04:32:48
# Regressor Bug: 1932150
# File Overlap Count: 1
# Description:
#   Revert "Bug 1932150 - Fix build bustage" for causing build bustages.
#   
#   This reverts commit 0681b181d6a8a48ac460a97dacb7fca91f9989c3.
#   
#   Revert "Bug 1932150 - Fix lint failure" for causing Build bustage.
# ==============================================================================

diff -r 1e64a48e3517 -r 4f30d7afdf22 dom/base/Selection.h
--- a/dom/base/Selection.h	Wed May 07 20:11:46 2025 +0300
+++ b/dom/base/Selection.h	Wed May 07 20:37:37 2025 +0300
@@ -934,8 +934,11 @@
       PostContentIterator& aPostOrderIter, nsIContent* aContent,
       bool aSelected) const;
 
-  void SelectFramesOfFlattenedTreeOfContent(nsIContent* aContent,
-                                            bool aSelected) const;
+  /**
+   * https://dom.spec.whatwg.org/#concept-shadow-including-descendant
+   */
+  void SelectFramesOfShadowIncludingDescendantsOfContent(nsIContent* aContent,
+                                                         bool aSelected) const;
 
   nsresult SelectFrames(nsPresContext* aPresContext, AbstractRange& aRange,
                         bool aSelect) const;